examples/kupo/docker-compose.yaml

```diff
@@ -23,12 +29,6 @@ services:
     environment:
       CARDANO_NODE_FLAG: ${CARDANO_NODE_FLAG}
       CONTAINER_SOCKET_PATH: ${CONTAINER_SOCKET_PATH} 
-    healthcheck:
-      test: /bin/healthcheck.sh
-      interval: 10s
-      timeout: 20s
-      retries: 30
-      start_period: 30s
```

Comment:
Why has this been removed now?

Reply:
See comment [below](https://github.com/mlabs-haskell/cardanow/pull/82#discussion_r1704104131)